src/outputs/outputs_main.py

```diff
@@ -58,14 +59,33 @@ def run_outputs(
 
     imputed_statuses = ["TMI", "CF", "MoR"]
 
-    to_keep = estimated_df["imp_marker"].isin(imputed_statuses) | (estimated_df["imp_marker"] == "R")
+    to_keep = estimated_df["imp_marker"].isin(imputed_statuses) | (
+        estimated_df["imp_marker"] == "R"
+    )
+    filtered_out = ~(
+        estimated_df["imp_marker"].isin(imputed_statuses)
+        | (estimated_df["imp_marker"] == "R")
+    )
```

Comment:
This can be simplified to 

filtered_out = ~(to_keep)

Or even simpler, you don't need the extra variable, just put 

filtered_output_df = estimated_df.copy().loc[~to_keep]

Reply:
Thanks @AnneONS, much neater!